resources/views/navbar-notifications.blade.php

```diff
@@ -8,12 +8,12 @@
                         :type="$notification->data['type']"
                     />
                     <div class="flex flex-col w-full ml-5 space-y-1">
-                        <div class="flex flex-row justify-between sm:space-x-2">
-                            <span class="font-semibold text-theme-secondary-900 sm:truncate">
+                        <div class="flex flex-row justify-between">
+                            <span class="font-semibold text-theme-secondary-900 @if(strlen($notification->name()) > 32)notification-truncate @endif">
```

Comment:
why is the check needed 🤔 can't it always have the class and then it simply doesn't truncate if there's enough space to fit the whole text

Reply:
Tried a couple of options, with both `truncate` and `notification-truncate` and the best results I had looking wise are that way. Issue I had for example by using `notification-truncate` without the check was that, a 2 letters name would be truncated, tried to have a second css class like `notification-title-truncate` and tweak it a bit but meh.